mlflow/models/wheeled_model.py

```diff
@@ -213,12 +215,46 @@ def _download_wheels(cls, pip_requirements_path, dst_path):
         Args:
             pip_requirements_path: Path to requirements.txt in the model directory
             dst_path: Path to the directory where the wheels are to be downloaded
+            extra_envs: Extra environment variables to be passed to the subprocess.
```

Comment:
What is the motivation of adding `extra_envs` option? Is this method expected to be called by users?

Reply:
Yes, if users want to use some more complex options not in the list, they can pass through extra_envs